modules/Bio/EnsEMBL/Compara/PipeConfig/HomologyAnnotation_conf.pm

```diff
@@ -67,6 +68,11 @@ sub default_options {
         'ref_dump_dir' => $self->o('ref_member_dumps_dir'),
         # Directory for diamond and fasta files for query genome
         'members_dumps_dir' => $self->o('dump_path'),
+        # Compara schema file path
+        'schema_file' => $self->check_file_in_ensembl('ensembl-compara/sql/table.sql'),
+        # Path to db_cmd.pl script
+        'hive_root_dir' => $ENV{'EHIVE_ROOT_DIR'},
+        'db_cmd_path'   => $self->o('hive_root_dir').'/scripts/db_cmd.pl',
```

Comment:
I was thinking to make this more robust by doing the following:
1. Add `check_exe_in_hive` method in `ComparaGeneric_conf` (`$self->o('hive_root_dir')` exists)
2. Add this command in `ENV.pm` inside `executable_locations` (`db_cmd_exe`?)
3. Replace these lines by `'db_cmd_exe' => $self->o('db_cmd_exe')`

Although I'm not 100% sure about point 1, since I don't think it will be used frequently.

Reply:
It will probably be used throughout the rapid release :shrug: